build.sc

```diff
@@ -844,9 +844,23 @@ trait CliIntegration extends SbtModule with ScalaCliPublishModule with HasTests
       def test(args: String*) = {
         val argsTask = T.task {
           val launcher = launcherTask().path
+          val debugReg = "--debug(:[0-9]+)?".r
+          val debugOpt = args.filter(a => debugReg.pattern.matcher(a).matches).headOption
+          val (isDebug, debugPort) = debugOpt match {
```

Comment:
It seems we could have only one variable here: `debugPortOpt` say (typed as `Option[String]` I guess).

When it's `None`, debugging is disabled, when it contains a value, it's the debug port. And we only set `test.scala-cli.debug.port` for debugging (no need of `test.scala-cli.debug.is-debug`), when `debugPortOpt` is non empty.